features/step_definitions/smokey_steps.rb

```diff
@@ -332,3 +332,9 @@ def get_status_code
   yield
   $proxy.har.entries.first.response.status
 end
+
+def normalise_path(path_str)
+  return path_str if path_str == "/"
+  return path_str[0..-2] if path_str[-1] == "/" # strip trailing slash
```

Comment:
This will allow for a false positive [over here](https://github.com/alphagov/smokey/blob/main/features/gov_uk.feature#L12).

One alternative would be to have a separate step that's more permissive - could be specific to this scenario or the other one.

Reply:
Thanks @benthorner, that is a great catch, I appreciate your keen 👀. Serves me right for not running the whole suite in my testing. Your suggestion should work.